src/test/java/no/unit/nva/model/util/OrgNumberMapperTest.java

```diff
@@ -0,0 +1,29 @@
+package no.unit.nva.model.util;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.net.URI;
+
+import static no.unit.nva.model.util.OrgNumberMapper.NO_MAPPING_ERROR_MESSAGE;
+
+public class OrgNumberMapperTest {
+
+    public static final String VALID_CRISTIN_ID = "https://api.cristin.no/v2/institutions/20202";
+    public static final String VALID_ORG_NUMBER = "919477822";
+    public static final String INVALID_ORG_NUMBER = "123";
+
+    @Test
+    public void testOrgNumberExistingInMapReturnsCristinId() {
```

Comment:
the "test" prefix is not necessary. We are in a test class with the `@Test` tag over the method